server/src/main/java/org/elasticsearch/health/GetHealthAction.java

```diff
@@ -124,22 +124,22 @@ public String toString() {
     }
 
     public static class Request extends ActionRequest {
-        private final boolean computeDetails;
         private final String componentName;
         private final String indicatorName;
+        private final boolean includeDetails;
```

Comment:
I believe this should be named `explain` too? 

Together with the entire concept of `computeDetails` - https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/health/HealthService.java#L90 (and onwards)

I think dropping the `include...` and `compute...` concepts would simplify things.

What do you think?

Reply:
I had intentionally isolated the change to `explain` to the REST layer because it just doesn't make as much sense to me as include/computeDetails.

Reply:
Alright, I've change includeDetails and compuateDetails to explain everywhere.